lib/shopping_cart.rb

```diff
@@ -0,0 +1,22 @@
+class ShoppingCart
+  attr_reader :name,
+              :capacity
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+  end
+
+  def capacity
+    @capacity.chomp("items")
+    @capacity.to_i
+  end
+
+  def products
+    @products = []
+  end
```

Comment:
this is something I think would be good in your initialize method since every Cart you make, you want it to start with an empty array of products.